src/tespy/networks/network.py

```diff
@@ -2698,7 +2698,7 @@ def print_results(self, colored=True, colors={}, return_result=False):
                         )
 
         # connection properties
-        df = self.results['Connection'].loc[:, ['m', 'p', 'h', 'T']]
+        df = self.results['Connection']
```

Comment:
There will be some small adjustment needed: Without unselecting the fluid names, there will be an error later in the displaying the results in line 2709.

Reply:
Thank you - we can just roll this commit back.